### DISCUSSION TITLE
Baked in browser tab-management and Tab-fern

### DISCUSSION THREAD

**Discussion by tscahill** [cxw42/TabFern#273] [General]
Recent versions of Firefox and Chrome have a new feature for managing tabs. In Firefox they are called Container Tabs, and in Chrome they are called Groups. Brave isn't supporting this feature, yet. I wonder if this nested tab feature the browsers are using could be leveraged by Tabfern. There might be something there that suggests how multi-level folders could be implemented.

**Comment by cxw42** [comment]
Yes, indeed --- #246 (Chrome) and #255 (Vivaldi) track these.  Firefox is blocked on general FF support (#200).
